package.json

```diff
@@ -211,56 +210,55 @@
         {
           "command": "extension.addObjectExplorer",
           "title": "%extension.addObjectExplorer%",
-          "group": "navigation",
-          "when": "enablePreviewFeatures"
+          "group": "navigation"
         }
       ],
       "view/item/context": [
         {
           "command": "extension.objectExplorerNewQuery",
-          "when": "enablePreviewFeatures && viewItem =~ /^(disconnectedServer|Server|Database|Table)$/",
+          "when": "viewItem =~ /^(disconnectedServer|Server|Database|Table)$/",
           "group": "MS_SQL@1"
         },
         {
           "command": "extension.removeObjectExplorerNode",
-          "when": "enablePreviewFeatures && viewItem =~ /^(disconnectedServer|Server)$/",
+          "when": "viewItem =~ /^(disconnectedServer|Server)$/",
           "group": "MS_SQL@3"
         },
         {
           "command": "extension.refreshObjectExplorerNode",
-          "when": "enablePreviewFeatures && view == objectExplorer",
+          "when": "view == objectExplorer",
           "group": "MS_SQL@2"
         },
         {
           "command": "extension.disconnectObjectExplorerNode",
-          "when": "enablePreviewFeatures && viewItem == Server",
+          "when": "viewItem == Server",
           "group": "MS_SQL@4"
         },
         {
           "command": "extension.scriptSelect",
-          "when": "enablePreviewFeatures && viewItem == Table"
+          "when": "viewItem == Table"
         }
       ],
       "commandPalette": [
         {
           "command": "extension.objectExplorerNewQuery",
-          "when": "enablePreviewFeatures && view == objectExplorer"
+          "when": "view == objectExplorer"
         },
         {
           "command": "extension.removeObjectExplorerNode",
-          "when": "enablePreviewFeatures && viewItem != null"
+          "when": "viewItem != null"
         },
```

Comment:
So I assume this is supposed to only show up when an OE item is selected? Did you verify that works? Although shouldn't this only be for server connection nodes? In general I would expect these when clauses to match the corresponding context contributions.